### PR TITLE
Self service runbooks for operations teams

### PR DESCRIPTION
**Pull request by adrianhowchin**
# About

This is a post about looking at runbooks from the perspective of an Operations Team.

# Pre-requisites

- [ ] The draft is complete and this post is ready to be reviewed.
- [ ] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

**Review by robpearson — COMMENTED** [review]
I think your post is excellent and I love that it is from the operations perspective. Something we’re sorely missing. 

I’ve made some suggestions to help improve the focus and clarity. I don’t think they’re drastic changes but smaller ones to help it shine. 

Shout out if you have any questions.

## Intro

Original: 
In this post, we will look from an Operations Team perspective at what a runbook is, what we mean by self-service runbooks, why you would want to use self-service runbooks, and how they help both you and your users.

Suggestion:
In this post, we look at the operations persectivte on what a runbook is, what we mean by self-service runbooks, and why self-service is so beneficial to both development and operations teams. 

## What are runbooks?
I would mention that runbooks are traditionally manual but runbook automation brings the benefit of DevOps thinking to the processes.


## What do we mean by self-service runbooks?
I’d suggest mentioning that automation is the key to this. 

## How do self-service runbooks benefit Operations Teams?
I’d suggest changing this from a how to a why. I think your text matches the Why question and that can help explain the value slightly better. 

I would tweak the headlines slightly. Not sure there’s are perfect though.

* Why self-service is beneficial to both development and operations teams? 
	* Example: Self-service runbooks can help to reduce shadow IT
	* Example: Self-service runbooks can help resolve production outages.

I’ve tweaked the second example (I’m calling them examples but I’m not married to the name) to focus on how self-service runbooks as that’s the core topic of the post. You could probably expand on this section a bit more to say how you can craft runbooks from past outages to mitigate known issues or sporadic problems fast. Combing monitoring with a process to identify the problem and then execute a runbook is super powerful and fast.

File: blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Self-service runbooks for Operations Teams
```
Comment (robpearson) [inline blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md:2]: I wonder if we should tweak this too. "Why self-service runbooks are good for operations teams" or something like that. :)

Comment (adrianhowchin) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md:2]: Something that shows a benefit to Ops teams would be good.

"Reduce shadow IT and improve uptime with self-service runbooks"
"Reduce your pile of work and improve uptime with self-service runbooks"
"Operations teams that use self-service runbooks get all the girls/boys" 
.. or something like that.

Comment (robpearson) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md:2]: Based on your feedback, I really like this? Thoughts?

> Reduce shadow IT and improve uptime with self-service operations

**Review by robpearson — APPROVED** [review]

File: blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+"Shadow IT" is the unapproved use of IT systems and applications within an organisations, and it is a big issue for many companies. Self-service runbooks can help to reduce shadow IT, as users are more likely to follow the right path when they can do so quickly and easily. From MRC:
+
+```
+One more thing: The goal of this step is controlled, self-service solutions. Any software you provide must meet two important criteria:
+
+- Self-service: Users must use the solution without bothering IT.
+- Control: IT must still be able to control data and user access.
+
+When you deliver controlled, self-service options, your business gets the best of both worlds. Users get the solutions they need quickly, and IT can still secure the data and applications."
+```
```
Comment (jedlehmann-prog) [inline blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md:45]: Should this be a `blockquote` rather than `code`?

Comment (wordlee) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md:45]: Yeah. I'll swap that out with the line edits :)

Comment (wordlee) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md:45]: I'll also drop `One more thing` from the beginning of the quote as out of context that phrase doesn't convey anything.

File: blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md (hunk 1 above)
Comment (wordlee) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md:2]: I think the original title fits:
> Self-service runbooks for operations teams
Shadow IT is only part of the post's focus. However, I think a post dedicated to the topic would be a great addition, and we already have the title!

**Review by wordlee — APPROVED** [review]

File: blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md (hunk 1 above)
Comment (robpearson) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md]: @leecarlon I agree but I think the Shadow IT angle helps make it more interesting. It's a different perspective on the self-service story we've been telling. 

What if we said "Using Self-service operations to reduce shadow IT and improve uptime" ? :) /cc @jedlehmann-prog

Comment (adrianhowchin) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md]: Agreed, I'd love to do a standalone post on how using Octopus can help reduce shadow IT. 

Sticking with the original it is!

File: blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md (hunk 2 above)
Comment (adrianhowchin) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md]: I'm not going to lie - I just copied and pasted Matt's post from November, and took a guess at the syntax. Don't assume I know what I'm doing here... :S

File: blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md (hunk 1 above)
Comment (jedlehmann-prog) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md]: I personally think the shadow IT title highlights the benefit and is more engaging. 

We also have a good image concept for the shadow IT angle so I could be biased 🙂

Comment (jedlehmann-prog) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md]: OK, let's go with the original title – I will work on an image for the post.

File: blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md (hunk 2 above)
Comment (wordlee) [reply blog/2020-12/self-service-runbooks-from-an-operations-perspective/index.md]: All good. I had to search previous posts for an example that worked :)

**Pull request merged**